zuul.d/whitebox_neutron_tempest_jobs.yaml

```diff
@@ -137,6 +137,10 @@
             excludeList: |
               # remove when bug OSPRH-9569 resolved
               ^whitebox_neutron_tempest_plugin.tests.scenario.test_metadata_rate_limiting
+              # remove when bug OSPRH-7998 resolved
```

Comment:
It could be nice to have also the reference of https://issues.redhat.com/browse/OSPRH-9203. This is the current issue that is causing this problem.